### PR TITLE
Fix lentperc yearly calculation

### PR DESCRIPTION
**Pull request by rnevet**
Old calculation would factor the lent percentage (lent/total) into the compound function, which led to lower yearly rate.

## TESTING STAGE
In progress.

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] **I have read CONTRIBUTING.md**
- [X] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [X] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [N/A] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [N/A] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [ ] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by rnevet** [comment]
@yura-pakhuchiy  can you review?

**Comment by yura-pakhuchiy** [comment]
LGTM

File: www/lendingbot.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
     <link href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.css" rel="stylesheet">
 
-    <script src="lendingbot.js?version=1" ></script>
+    <script src="lendingbot.js?version=2" ></script>
```
Comment (yura-pakhuchiy) [inline www/lendingbot.html]: Why do we need this? Ctrl-F5 in browser after upgrade should handle reload without cache. Incrementing it manually after every edit a bit overkill IMHO.

Comment (rnevet) [inline www/lendingbot.html]: You are right, it's just a way for the dev to ensure a refresh.

**Pull request merged**